### PR TITLE
IBX-6386: DAM and image management

### PR DESCRIPTION
**Pull request by juskora**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | https://issues.ibexa.co/browse/IBX-6386
| Versions      | 4.6, master
| Edition       | all

Screenshots will be added when the final UI is ready.

#### Checklist

- [ ] Text renders correctly
- [ ] Text has been checked with vale
- [ ] Description metadata is up to date
- [ ] Redirects cover removed/moved pages
- [ ] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

File: docs/dam/ibexa_dam.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+edition: headless
+---
+
+# Ibexa DAM
+
+Digital Asset Management is a platform dedicated to editors. It enables storing in central location, organizing, distributing, 
+and sharing media assets across many channels.
+
+[[= product_name_dam =]] image picker is developed to work with any web browser that supports modern standards. However for the best user experience in editorial work
+it's recommended to use 1366x768 screen resolution.
```
Comment (ciastektk) [inline docs/dam/ibexa_dam.md:12]: Are we sure about this recommended screen resolution? Isn't this the minimum resolution?

File: docs/dam/ibexa_dam.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+- language
+- format
+- file size
+- orientation
+- dimensions
+- tags (this filter is available only when image assets have tags assigned to)
+- date of creation
+
+To use it, in the right-side panel select attributes and click the **Apply** button.
+You can combine searches by all types of filtering: keyword search and attributes filter.
```
Comment (ciastektk) [inline docs/dam/ibexa_dam.md:62]: We should add very important information that image filtering works only when DXP uses `solr` or `elasticsearch` search engine. LSE (Legacy Search Engine) is not supported.

File: docs/image_management/upload_images.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+5\. In the **Asset** section, click **Upload file** and from the system browser, select an image.
+
+6\. To find images using tags, in the **Tags** field, enter keywords.
+
+Tagging image assets helps keep the media library organized and enables searching images by tags in the DAM.
+
+7\. Click **Publish**.
+
+!!!note 
+    The maximum file size and file extension limitations may differ depending on your configuration.
```
Comment (ciastektk) [inline docs/image_management/upload_images.md:51]: Have we described how to limit image file extensions? 

Screenshot 2024-01-09 at 09 47 44

File: docs/dam/ibexa_dam.md (hunk 1 above)
Comment (juskora) [reply docs/dam/ibexa_dam.md:12]: It's taken from spec docs :)

Comment (ciastektk) [reply docs/dam/ibexa_dam.md:12]: Not really. Spec describes minimum screen resolution. IMO, minimum is not the same as recommended.

File: docs/dam/ibexa_dam.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+### Grid view
+
+Grid view contains image preview, file format, and image size.
+
+### List view
+
+List view contains a thumbnail, title, format, size, dimensions, creation, and update date.
+
+### Sorting
```
Comment (julitafalcondusza) [inline docs/dam/ibexa_dam.md:34]: I would move it one level higher, not inside of "View modes" section. :)

```suggestion
## Sorting
```

**Review by julitafalcondusza — APPROVED** [review]

**Pull request merged**